src/lib_c/x86_64-windows-msvc/c/minwinbase.cr

```diff
@@ -45,6 +45,9 @@ lib LibC
     GetFileExMaxInfoLevel
   end
 
+  LOCKFILE_FAIL_IMMEDIATELY = DWORD.new(0x00000001)
+  LOCKFILE_EXCLUSIVE_LOCK   = DWORD.new(0x00000002)
```

Comment:
This is not really needed, right?
```suggestion
  LOCKFILE_FAIL_IMMEDIATELY = 0x00000001
  LOCKFILE_EXCLUSIVE_LOCK   = 0x00000002
```

Reply:
Not necessarily, but that's only because `DWORD` is equivalent to the default integer type `Int32`.
Prior evidence: 

https://github.com/crystal-lang/crystal/blob/022e1f9fe3d57b93f841789ea0bc477938bc8abe/src/lib_c/x86_64-windows-msvc/c/fileapi.cr#L10-L13